packages/server/routes/claimReview.js

```diff
@@ -1,6 +1,7 @@
 'use strict';
 
-const claimReview = require('../api/controller/claimReviewController');
+const ClaimReview = require('../api/controller/claimReviewController');
```

Comment:
For consistency sake, rename ClaimReview to ClaimReviewController. Apply this everywhere for other classes.

Reply:
ok.